module-data/src/main/java/org/lindeck/data/model/cards/CardWithInput.java

```diff
@@ -0,0 +1,35 @@
+package org.lindeck.data.model.cards;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+
+/**
+ * @author derovi
+ * This card has input field, so user answer will be validated.
+ */
+
+@EqualsAndHashCode(callSuper = true)
+@Entity
+@Data
+public class CardWithInput extends AbstractCard {
+    public enum AnswerType {
+        INTEGER, FLOAT, STRING
+    }
```

Comment:
why you split the answer by type? I think it is ok, just to keep the answer only in a string,
Seems to me as an unnecessary complication